Runtime/AvatarCreator/Scripts/WebRequests/Responses/GetUserAvatarsResponse.cs

```diff
@@ -0,0 +1,11 @@
+using ReadyPlayerMe.Core;
+
+namespace ReadyPlayerMe.AvatarCreator.Responses
+{
+    public struct GetUserAvatarResponse
```

Comment:
is the "Get" really necessary for this struct type name? 
It seems a little misleading as you usually use Get for a function that retrieves a value. 
EG GetAvatarID. 
I would say for a struct or type name UserAvatarResponse would be better.